db/models.py

```diff
@@ -1 +1,50 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.description}"
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             on_delete=models.CASCADE,
+                             related_name="skill_set")
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.bonus} {self.race}"
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.description}"
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             on_delete=models.CASCADE,
+                             related_name="players_race")
+    guild = models.ForeignKey(Guild,
+                              on_delete=models.SET_NULL,
+                              null=True, blank=True,
+                              related_name="players_guild")
```

Comment:
Related name is selected incorrectly, leave only `players` to call them from `Guild`. Apply the same change to the race field as well.